ios/Classes/SwiftVitalFlutterPlugin.swift

```diff
@@ -4,11 +4,15 @@ import Combine
 import VitalCore
 import VitalHealthKit
 
-public class SwiftVitalFlutterPlugin: NSObject, FlutterPlugin {
+public class SwiftVitalFlutterPlugin: NSObject, FlutterPlugin, FlutterApplicationLifeCycleDelegate {
```

Comment:
`@protocol FlutterPlugin <NSObject, [FlutterApplicationLifeCycleDelegate]>`

You don't need the extra compliance.